src/main/io/vtx_smartaudio.c

```diff
@@ -447,9 +454,8 @@ static void saSendFrame(uint8_t *buf, int len)
         serialWrite(smartAudioSerialPort, buf[i]);
     }
 
-    if (vtxSettingsConfig()->akkStyleEndFrame) {
-      serialWrite(smartAudioSerialPort, 0x00); // Added back to fix old AKK - breaks custom freq.
-    }
+	serialWrite(smartAudioSerialPort, 0x00); // Re-added for both "Legacy" and "Standard" SA implementations
```

Comment:
I am really hoping this is going to solve some of that weirdness.. though we should look more into the software pulldown.